docs/synapse/userguides/storm_ref_data_mod.rstorm

```diff
@@ -0,0 +1,746 @@
+
+Incorrectly specifying a property value is unlikely to occur for users entering Storm data modification queries at the command line (barring outright user error), as users are directly vetting the data they are entering. However, the try operator may be useful for Storm-based automated ingest of data (such as :ref:`syn-tools-csvtool` or :ref:`syn-tools-feed`) where the data source may contain "bad" data.
+
+Use of the try operator allows Storm to fail silently in the event it encounters a ``BadTypeValu`` error (i.e., skip the bad event but continue processing). Contrast this behavior with using the standard equivalent operator ( ``=`` ), where if Storm encounters an error it will halt processing.
+
+See the :ref:`type-array` section of the :ref:`storm-ref-type-specific` for specialized "edit try" syntax when working with arrays.
+
+Autoadds and Depadds
+++++++++++++++++++++
+
+Synapse makes use of two optimization features when adding nodes or setting secondary properties: automatic additions (:ref:`gloss-autoadd`) and dependent additions (:ref:`gloss-depadd`).
+
+**Autoadd** is the process where, on node creation, Synapse will automatically set any secondary properties that are derived from a node's primary property. Because these secondary properties are based on the node's primary property (which cannot be changed once set), the secondary properties are read-only.
+
+**Depadd** is the process where, on setting a node's secondary property value, if that property is of a type that is also a form, Synapse will automatically create the form with the corresponding primary property value. (You can view this as the secondary property "depending on" the existence of a node with the corresponding primary property.)
+
+Autoadd and depadd work together (and recursively) to simplify adding data to a Cortex. Properties set via autoadd may result in the creation of nodes via depadd; the new nodes may have secondary properties set via autoadd that result in the creation of additional nodes via depadd, and so on.
+
+
+**Examples:**
+
+.. NOTE::
+  The specific syntax and process of node creation, modification, etc. are described in detail below. The examples here are simply meant to illustrate the autoadd and depadd concepts.
+
+*Create a node for the email address user@vertex.link. Note the secondary properties (:fqdn and :user) that are set via autoadd.*
+
+
+.. storm-cli:: [ inet:email = user@vertex.link ]
+
+
+*Create a node to represent the twitter account for The Vertex Project. Synapse creates the account itself (``inet:web:acct``) with secondary properties for ``:webpage`` (explicitly set) as well as ``:site`` and ``:user`` (via autoadd). Note the additional nodes that are created from those secondary properties via deppadd (``inet:url``, ``inet:user``, multiple FQDNs, etc.).*
```

Comment:
Is this supposed to be italicized text?

Reply:
it looks like this is consistent with the rest of the docs